gems/aws-sdk-core/lib/aws-sdk-core/plugins/sign.rb

```diff
@@ -97,7 +97,7 @@ def initialize(auth_scheme, config, region_override = nil)
           end
 
           region = if scheme_name == 'sigv4a'
-                     auth_scheme['signingRegionSet'].first
+                     auth_scheme['signingRegionSet'].join(',')
```

Comment:
This should work with the sigv4 signer, but its a little confusing.  Maybe we should at least update the docs in the sigv4 signer to specify that for sigv4a region is the regionSet as a comma seperated list.